### ISSUE TITLE
Question: is it possible to monitor stack/variables on the go?

### ISSUE THREAD

**Issue by grappas** [rcarriga/nvim-dap-ui#338]
Instead only on breakpoint or step. I don't want to always stop the program to analyze variables.

**Comment by igorlfs** [comment]
You can use conditional breakpoints. Does that fit your use case?

**Comment by grappas** [comment]
No, it's not. It could be a breakpoint that would be automatically resumed. I want to check how data is processed through strict timeline. In other words it cannot be stopped all together.

**Comment by sleeptightAnsiC** [comment]
Perhaps `DAP Watches` is what you were looking for @grappas ? 
https://github.com/rcarriga/nvim-dap-ui/blob/e94d98649dccb6a3884b66aabc2e07beb279e535/README.md?plain=1#L105-L123

**Comment by grappas** [comment]
> Perhaps `DAP Watches` is what you were looking for @grappas ?
> 
> https://github.com/rcarriga/nvim-dap-ui/blob/e94d98649dccb6a3884b66aabc2e07beb279e535/README.md?plain=1#L105-L123

That's **where** I want that feature.

**Comment by sleeptightAnsiC** [comment]
I think, there is something like this already implemented in `nvim-dap`
It's called "LogPoint" and you set it with `dap.set_breakpoint` like so:
```lua
require('dap').set_breakpoint(nil, nil, "i: {i}\n")
```
Then it should log into either `DAP Repl` or `DAP Console` depending on Adapter. 
For example, this is how it looks like with native GDB and simple Rust code:

![image](https://github.com/user-attachments/assets/9d0771d7-4500-40ba-b102-9f37b6f3c199)

The program never stops. It just runs and prints the values.
See `:h dap.set_breakpoint()`

Is this what you were looking for? @grappas

**Comment by grappas** [comment]
> Is this what you were looking for? @grappas
That's the step forward, but not exactly what I've been hoping for.
a) You have to manually define b) an exact variable/s
c) You have to sift it through log which is not so convenient

Don't get me wrong - your method will change my life forever ( ͡° ͜ʖ ͡°), but imagine: don't think about it, add watch to the scope your interested in and just watch all of it on the left of the window. It would be super convenient.
